src/app/zap-templates/zcl/data-model/chip/microwave-oven-control-cluster.xml

```diff
@@ -32,27 +32,27 @@ limitations under the License.
     <attribute side="server" code="0x0001" define="MAX_COOK_TIME"       type="elapsed_s"               writable="false" optional="false" min="1" max="86400">MaxCookTime</attribute>
-    <attribute side="server" code="0x0002" define="POWER_SETTING"       type="int8u"     default="100" writable="false" optional="true">PowerSetting</attribute>
-    <attribute side="server" code="0x0003" define="MIN_POWER"           type="int8u"     default="10"  writable="false" optional="true">MinPower</attribute>
-    <attribute side="server" code="0x0004" define="MAX_POWER"           type="int8u"     default="100" writable="false" optional="true">MaxPower</attribute>
-    <attribute side="server" code="0x0005" define="POWER_STEP"          type="int8u"     default="10"  writable="false" optional="true">PowerStep</attribute>
-    <attribute side="server" code="0x0006" define="SUPPORTED_WATTS"     type="array" entryType="int16u" writable="false" optional="true">SupportedWatts</attribute>
-    <attribute side="server" code="0x0007" define="SELECTED_WATT_INDEX" type="int8u"                   writable="false" optional="true">SelectedWattIndex</attribute>
+    <attribute side="server" code="0x0002" define="POWER_SETTING"       type="int8u"     default="100" writable="false" optional="true"  min="1" max="100">PowerSetting</attribute>
+    <attribute side="server" code="0x0003" define="MIN_POWER"           type="int8u"     default="10"  writable="false" optional="true"  min="1" max="100">MinPower</attribute>
+    <attribute side="server" code="0x0004" define="MAX_POWER"           type="int8u"     default="100" writable="false" optional="true"  min="1" max="100">MaxPower</attribute>
+    <attribute side="server" code="0x0005" define="POWER_STEP"          type="int8u"     default="10"  writable="false" optional="true"  min="1" max="100">PowerStep</attribute>
+    <attribute side="server" code="0x0006" define="SUPPORTED_WATTS"     type="array" entryType="int16u" writable="false" optional="true" min="1" max="10">SupportedWatts</attribute>
+    <attribute side="server" code="0x0007" define="SELECTED_WATT_INDEX" type="int8u"                   writable="false" optional="true"  min="0" max="9">SelectedWattIndex</attribute>
     <attribute side="server" code="0x0008" define="WATT_RATING"         type="int16u"                  writable="false" optional="true">WattRating</attribute>
 
     <command source="client" code="0x00" name="SetCookingParameters" optional="false">
         <description>Set Cooking Parameters</description>
         <arg name="CookMode"          type="int8u"     optional="true"/>
-        <arg name="CookTime"          type="elapsed_s" optional="true"/>
-        <arg name="PowerSetting"      type="int8u" min="MIN_POWER" max="MAX_POWER" optional="true"/>
-        <arg name="WattSettingIndex"  type="int8u"     optional="true"/>
+        <arg name="CookTime"          type="elapsed_s" min="1" max="MAX_COOK_TIME"     optional="true"/>
```

Comment:
Here also min/max are ignored, which is good, because nothing would know what to do with that max value....